Rubberduck.Core/UI/CodeExplorer/Commands/AddTemplateCommand.cs

```diff
@@ -4,11 +4,12 @@
 using System.Runtime.InteropServices;
 using Rubberduck.Navigation.CodeExplorer;
 using Rubberduck.Templates;
+using Rubberduck.UI.Command;
 using Rubberduck.VBEditor.SafeComWrappers;
 
 namespace Rubberduck.UI.CodeExplorer.Commands
 {
-    public class AddTemplateCommand : CodeExplorerCommandBase
+    public class AddTemplateCommand : CommandBase
```

Comment:
TBH, it feels a bit weird to make it a generic Command when it's a part of the code explorer. 

I think it can be made compatible with the `CodeExplorerCommandBase` by changing `ApplicableNodes` to something else and have it contain only one member for the tuple type. The additional conditions from the original `ApplicableNodes` (renamed to something else) can be then added instead.

Reply:
This will require some investigation with the debugger. After all the _Node_ in `ApplicableNodeTypes` is not really true. I will just have to find out the appropriate type and maybe change slightly how the check works.